.evergreen/Dockerfile.erb

```diff
@@ -182,6 +182,18 @@ WORKDIR /app
 
 <% end %>
 
+<% if fle? %>
+  RUN curl -fLo libmongocrypt-all.tar.gz "https://s3.amazonaws.com/mciuploads/libmongocrypt/all/master/latest/libmongocrypt-all.tar.gz"
+  RUN tar xf libmongocrypt-all.tar.gz
+
+  <% if libmongocrypt_path %>
+    ENV LIBMONGOCRYPT_PATH <%= libmongocrypt_path %>
+  <% else %>
```

Comment:
Perhaps this `else` can become a `rescue` if the lmc_path method is changed to raise.